BenadrylAutomation/src/main/java/com/jnj/pageobjects/BenadrylUsesPage.java

```diff
@@ -0,0 +1,291 @@
+		String actualDesc;
+		WebElement ele = null;
+
+		switch (symptom) {
+		case "Itchy Skin":
+			ele = this.getItchySkinSympEle(cause);
+			break;
+		case "Cold":
+			ele = this.getColdSympEle(cause);
+			break;
+		default:
+			BaseClass.extentInfoLog("Invalid case specified : " + symptom);
+		}
+		Action.scrollIntoCenterUsingJS(driver, ele);
+		actualDesc = Action.removeNonAlphanumericASCII(ele.getText());
+		Action.printAndAssert(actualDesc, Action.removeNonAlphanumericASCII(description));
+	}
+
+	/**
+	 * Function to verify links
+	 * 
+	 * @param linktext    pass linktext to click on
+	 * @param expectedUrl pass expected url
+	 */
+	public void verifyLink(String linktext, String expectedUrl) {
+		Action.scrollIntoCenterUsingJS(driver, this.getLinkByText(linktext));
+		Action.performActionwithExtentInfoLog(this.getLinkByText(linktext), "click",
+				"Clicked on : " + this.getLinkByText(linktext).getText());
+		Action.verifyPageUrl(expectedUrl);
+	}
+
+	/**
+	 * Function to verify headings
+	 * 
+	 * @param expectedHeading pass expected heading
+	 */
+	public void verifyheading(String expectedHeading) {
+		WebElement ele = null;
+		String currentHeading;
+		if (expectedHeading.contains("Product Should I Take")) {
+			ele = this.whichProductHeader;
+		} else if (expectedHeading.contains("How to Use BENADRYL")) {
+			ele = this.howToUseHeader;
+		} else if (expectedHeading.contains("for Cold Symptoms")) {
+			ele = this.forColdHeader;
+		} else if (expectedHeading.contains("for Itchy Skin Symptoms")) {
+			ele = this.itchySkinSympHeader;
+		} else {
+			BaseClass.extentFailLog("Invalid header expected : ", expectedHeading);
+		}
+		Action.scrollIntoCenterUsingJS(driver, ele);
+		expectedHeading = Action.removeNonAlphanumericASCII(expectedHeading);
+		currentHeading = Action.removeNonAlphanumericASCII(ele.getText());
+		Action.printAndAssert(currentHeading, expectedHeading);
+		BaseClass.extentInfoLog("Assertion success for header : " + currentHeading);
+	}
+
+	/**
+	 * Function to verify url from articles
+	 * 
+	 * @param heading     pass heading
+	 * @param article     pass article
+	 * @param readMore    pass read more
+	 * @param expectedUrl pass expected Url
+	 */
+	public void verifyArticles(String heading, String article, String readMore, String expectedUrl) {
+		Action.explicitWait(getSpanHeadings(heading), 30);
+		boolean eleDisplayed = getSpanHeadings(heading).isDisplayed();
+		if (eleDisplayed) {
+			extentPassLog("Related Content header displayed : ", true);
+			String actualHeading = getSpanHeadings(heading).getText();
+			Action.printAndAssert(actualHeading, heading);
+			Action.explicitWait(articleName(article), 30);
+			String actualArticle = articleName(article).getText();
+			Action.printAndAssert(actualArticle, article);
+			Action.explicitWait(readMoreLink(readMore), 30);
+			Action.performActionwithExtentInfoLog(readMoreLink(readMore), "click",
```

Comment:
@goutamnaik007 pls remove the empty line

Reply:
Removed @vaibhavNagvekarSJI